### PR TITLE
Disc 427 empty fields

### PR DESCRIPTION
**Pull request by VictorHarbo**
Corrections of XSLTs and solr schema

### CODE REVIEW

File: src/main/resources/xslt/schemaorg2solr.xsl [hunk 1]
```diff
@@ -450,6 +450,45 @@
...
+      <xsl:variable name="overlaps" as="item()*">
+        <xsl:copy-of select="array:flatten($internalMap('kb:program_structure_overlap'))"/>
+      </xsl:variable>
+
+      <!-- Defines fields for overlaps of type 1 and 2. Currently, I don't know if more than two types exists, '
+           then they would have to be added. -->
+      <xsl:for-each select="$overlaps">
+        <xsl:choose>
+          <xsl:when test="map:get(. , 'overlap_type') = '1'">
+            <f:string key="internal_program_structure_overlap_type_one_length_ms">
```
Comment (tokee) [inline src/main/resources/xslt/schemaorg2solr.xsl:465]: `internal_program_structure_overlap_type_one_length_ms` :grimacing: 
But okay, I get that we need to spell it put. Keep it as-is.

File: src/main/solr/dssolr/conf/schema.xml [hunk 2]
```diff
@@ -661,10 +661,33 @@ In solrconfig.xml the classic schema is defined with this:  <schemaFactory class
     <?description ?>
     <?example     ?>
   </field>
-  <!--TODO: Figure out what this field can contain-->
-  <field name="internal_program_structure_overlaps" type="string">
-    <?description ?>
-    <?example     ?>
+  <field name="internal_program_structure_overlaps" type="boolean">
+    <?description A boolean value that indicates if there are overlaps between this resource and other resources.?>
```
Comment (tokee) [inline src/main/solr/dssolr/conf/schema.xml:665]: What is an overlap? I'm guessing it is time based and probably coupled to wall time? Don't be afraid to use more words!

File: src/main/solr/dssolr/conf/schema.xml [hunk 3]
```diff
@@ -661,10 +661,33 @@ In solrconfig.xml the classic schema is defined with this:  <schemaFactory class
...
+    <?example     true?>
+  </field>
+  <field name="internal_program_structure_overlap_type_one_length_ms" type="plong">
+    <?description Indicates the length of an overlap of type one in ms.?>
```
Comment (tokee) [inline src/main/solr/dssolr/conf/schema.xml:669]: What is type one? As we don't know, please make it easier for future developers to investigate by stating what we /do/ know, i.e. that it is from the Preservica system.

**Review by tokee — COMMENTED** [review]
Looks like a job well done. Only minor comments about documentation.

File: src/main/resources/xslt/schemaorg2solr.xsl (hunk 1 above)
Comment (VictorHarbo) [reply src/main/resources/xslt/schemaorg2solr.xsl:465]: Not pretty. Lets remember to define which fields that are to be returned in a "normal" solr query.

File: src/main/solr/dssolr/conf/schema.xml (hunk 3 above)
Comment (tokee) [reply src/main/solr/dssolr/conf/schema.xml]: Very nice now!

**Pull request merged**